v2/charts/azure-service-operator/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: azure-service-operator
+version: v2.0.0-alpha.5
+description: Deploy components and dependencies of azure-service-operator
+home: https://github.com/Azure/azure-service-operator/v2
+sources:
+  - https://github.com/Azure/azure-service-operator/v2
+dependencies:
+  - name: aad-pod-identity
```

Comment:
@super-harsh  Also leaning towards option 2. Workload identity is a month out from having everything we need. What does migrating the chart entail?

Also are we listing this as a dependency because we anticipate users will use ASO with pod-identity/workload-identity, or are we using it for something else?

Reply:
@sakthi-vetrivel Dependency here is that the `dependency chart` should be present in the main Chart, however the dependency install flag will be inside the `Values.yaml` file which users can turn on and off. Default would be off.